execute/tokendata/usdc/attestation.go

```diff
@@ -30,15 +45,62 @@ type AttestationStatus struct {
 // Ethereum ->
 //
 //	12 ->
```

Comment:
Does this mean for `12` there is no entry in the map?

Reply:
It's nested map :D So 12 is a chainSelector, next line contains padded messageTokenIds